app/models/item.rb

```diff
@@ -103,4 +103,9 @@ def csv_export_attributes
       canonical_item.name
     ]
   end
+
+  def default_quantity
+    # TODO: actual logic for calculating and letting users configure this calculation
+    50
```

Comment:
:+1:

Reply:
Is there an issue to add the default_quantity to items?

Reply:
I'll make one

Reply:
https://github.com/rubyforgood/diaper/issues/785